### ISSUE TITLE
Stylistic set for Go

### ISSUE THREAD

**Issue by bebraw** [MonoLisaFont/feedback#59]
My understanding is that Go is well supported by ligatures already. Here's a complete set of characters to validate: https://golang.org/ref/spec#Operators_and_punctuation .

```
+    &     +=    &=     &&    ==    !=    (    )
-    |     -=    |=     ||    <     <=    [    ]
*    ^     *=    ^=     <-    >     >=    {    }
/    <<    /=    <<=    ++    =     :=    ,    ;
%    >>    %=    >>=    --    !     ...   .    :
     &^          &^=
```

**Comment by flowchartsman** [comment]
Heavy Go user here, would love to see some of these before taking the plunge.

**Comment by MarcusSterz** [comment]
Wondering if the second ligature is an option or too confusing? Otherwise `%=` would be only a white space ligature.
<img width="236" alt="Bildschirmfoto 2023-11-16 um 13 39 44" src="https://github.com/MonoLisaFont/feedback/assets/51958720/c7029379-f2c2-4551-8c2e-d45db3ba46c6">

**Comment by bebraw** [comment]
Whitespace alignment might be safer.

**Comment by okonet** [comment]
Yeah, the ligature looks weird.

**Comment by bebraw** [comment]
There's a prerelease containing Go specific ligatures available now. These might have to go behind a specific stylistic set in case they are too specific for other languages.

**Comment by bebraw** [comment]
Included in 2.012. The exact ligatures added are +=, -=, *=, %=, &=, &^, and &^=.
